docs/data/material/guides/nextjs/nextjs.md

```diff
@@ -59,6 +59,35 @@ This option ensures that the styles generated by Material UI will be wrapped in
 
 To learn more about it, see [the MDN CSS layer documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer).
 
+### Link
+
+The package provides a thin wrapper Link component that combines the functionality of the Next.js [Link](https://nextjs.org/docs/api-reference/next/link) and the styles of Material UI [Link](https://mui.com/components/links/).
+
+You can pass any props supported by both components to the Link.
+
+```js
+import { Link } from '@mui/material-nextjs/v13-appRouter'; // or `v14-appRouter` if you are using Next.js v14
+
+function Nav() {
+  return (
+    <div>
+      <Link href="/dashboard" variant="caption" sx={{ bgcolor: 'grey.100' }}>
+        Dashboard
+      </Link>
+    </div>
+  );
+}
+```
+
+To learn more about the supported props, visit:
+
+- Next.js related props: [Next.js Link](https://nextjs.org/docs/app/api-reference/components/link#props)
+- Material UI related props: [Material UI Link](/material-ui/react-link/)
+
+:::warning
+Do not use the Link from <b>`@mui/material-nextjs/*-pagesRouter`</b> for App Router.
+:::
+
```

Comment:
Is this needed?

```suggestion
```

Because we import the router, it will crash if it uses the wrong API

![Screenshot 2024-01-16 at 01 21 57](https://github.com/mui/material-ui/assets/3165635/aa356fb4-efd5-4ed1-93fb-857aac162836)